troposphere/wafv2.py

```diff
@@ -399,19 +399,13 @@ class Regex(AWSProperty):
     }
 
 
-class RegularExpressionList(AWSProperty):
-    props = {
-        'RegularExpressionList': ([Regex], False)
-    }
-
-
 class RegexPatternSet(AWSObject):
     resource_type = "AWS::WAFv2::RegexPatternSet"
 
     props = {
         'Description': (basestring, False),
         'Name': (basestring, True),
-        'RegularExpressionList': (RegularExpressionList, False),
+        'RegularExpressionList': ([Regex], True),
```

Comment:
The docs specify "List of String". Should this be "[basestring]" and remove "Regex"?

Reply:
Good point, I mistakenly assumed that the Regex object had some form of validation added. I had verified that it doesn't. The PR has been amended, thanks for reviewing.